pkg/devspace/config/versions/validate.go

```diff
@@ -256,8 +256,11 @@ func validateDeployments(config *latest.Config) error {
 		if deployConfig.Helm == nil && deployConfig.Kubectl == nil {
 			return errors.Errorf("Please specify either helm or kubectl as deployment type in deployment %s", deployConfig.Name)
 		}
-		if deployConfig.Kubectl != nil && deployConfig.Kubectl.Manifests == nil {
-			return errors.Errorf("deployments[%s].kubectl.manifests is required", index)
+		if deployConfig.Kubectl != nil && deployConfig.Kubectl.Manifests == nil && deployConfig.Kubectl.InlineManifest == "" {
+			return errors.Errorf("deployments[%s].kubectl.manifests or deployments[%s].kubectl.InlineManifest is required", index, index)
+		}
+		if deployConfig.Kubectl != nil && deployConfig.Kubectl.Manifests != nil && deployConfig.Kubectl.InlineManifest != "" {
+			return errors.Errorf("deployments[%s].kubectl and deployments[%s].inlineManifest cannot be used together", index, index)
```

Comment:
Should this message be the following?
```golang
"deployments[%s].kubectl.manifests and deployments[%s].kubectl.inlineManifest cannot be used together"
```

Reply:
You're right :sweat_smile: Fixed it now :+1: 

Will also work on the e2e :+1: